GiEnJul/Controllers/GiverController.cs

```diff
@@ -52,7 +52,14 @@ public async Task<ActionResult<PostGiverResultDto>> PostAsync([FromBody] PostGiv
 
             var insertedAsDto = _mapper.Map<PostGiverResultDto>(await _giverRepository.InsertOrReplaceAsync(giver));
 
-            var messageContent = $"{insertedAsDto.FullName}, vi bekrefter å ha mottatt din registering for å gi en jul i {insertedAsDto.Location}";
+            var messageContent =
```

Comment:
Should the text be equal to what they just saw on the website?

![image](https://user-images.githubusercontent.com/45152025/128479330-13c71cfe-6d70-49c8-a4e9-c00991c14738.png)

Reply:
Removed the `Tusen takk for innsatsen dere skal gjøre!` part. I think the rest is necessary.